charts/newrelic-infrastructure-v3/templates/control-plane/_helpers.tpl

```diff
@@ -0,0 +1,8 @@
+{{/* Returns whether the controlPlane scraper should run with hostNetwork: true based on the user configuration. */}}
+{{- define "newrelic.controlPlane.hostNetwork" -}}
+{{- if .Values.privileged -}}
+true
+{{- else if .Values.controlPlane.unprivilegedHostNetwork -}}
+true
```

Comment:
I would change the order, we what to honor `hostNetwork` value before the `privileged`.

Could this be another issue like the `lowDataMode` where we want to differentiate between `false` and "falsiness"?